src/main/java/com/logpresso/scanner/Log4j2Scanner.java

```diff
@@ -400,15 +402,19 @@ private void fix() {
 				if (lockError) {
 					reportError(f, "Cannot lock file " + t.getMessage(), t);
 				} else if (truncateError) {
-					backupFile.delete();
+					if(!backupFile.delete()) {
+					  System.out.println("backup file cannot be deleted");
```

Comment:
User should be notified which file was problem
`System.out.println("Error:  Backup file cannot be deleted - " + backupFile.getAbsolutePath());`